requirements.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.7
```

Comment:
Compiled with 3.7?

Reply:
yes, it's the earliest version we support. Without this we install newer library versions that will not work with 3.7